xray_proto_rust/src/proto.proto

```diff
@@ -23,9 +23,14 @@ message Vector2f {
   float y = 2;
 }
 
+message Vector2d {
+  double x = 1;
+  double y = 2;
+}
+
 message Rect {
-  Vector2f min = 1;
-  float edge_length = 2;
+  Vector2d min = 3;
```

Comment:
Hi, I am not so familiar with protos yet, why does this proto declaration begin with 3?

Reply:
Whenever a new field is added, the number it carries will be the last used one plus one.

Reply:
thanks for the explanation!